src/OpenLoco/src/Vehicles/OrderManager.cpp

```diff
@@ -110,6 +110,44 @@ namespace OpenLoco::Vehicles::OrderManager
         }
     }
 
+    bool spaceLeftInGlobalOrderTableForOrder(const Order* order)
+    {
+        return numOrders() + kOrderSizes[enumValue(order->getType())] <= Limits::kMaxOrders;
+    }
+
+    bool spaceLeftInVehicleOrderTable(VehicleHead* head)
+    {
+        auto ring = head->getCurrentOrders();
+        size_t size = std::distance(ring.begin(), ring.end());
```

Comment:
could you verify that this doesn't account for the end? The old code for a full routing table should result in a value of 63. I'm unsure if this will produce 63 or 64.

Reply:
This results in the same behaviour as master. On master, I am able to extend a routing table to 62 waypoints. I am able to do the same thing on this PR.